tests/api/test_attributes.py

```diff
@@ -98,6 +98,35 @@ def test_attributes_query(user_api_client, product):
     assert len(attributes_data) == attributes.count()
 
 
+NOT_EXISTS_IDS_ATTRIBUTES_QUERY = """
+    query ($filter: AttributeFilterInput!) {
+        attributes(first: 5, filter: $filter) {
+            edges {
+                node {
+                    id
+                    name
+                }
+            }
+        }
+    }
+    """
+
+
+@pytest.mark.parametrize(
+    "attribute_filter", [{"ids": ["ygRqjpmXYqaTD9r=", "PBa4ZLBhnXHSz6v="]}],
+)
+def test_attributes_query_ids_not_exists(attribute_filter, user_api_client, category):
+    query = NOT_EXISTS_IDS_ATTRIBUTES_QUERY
+    variables = {"filter": attribute_filter}
+    response = user_api_client.post_graphql(query, variables)
+    content = get_graphql_content(response, ignore_errors=True)
+    message_error = "{'ids': [{'message': 'Invalid ID specified.', 'code': ''}]}"
+
+    assert content["errors"]
```

Comment:
I would suggest doing the following:
```python
assert len(content["errors"]) == 1
```

This allow to ensure there are errors, and actually the amount that we are expecting

Reply:
working on it. thanks :)

Reply:
Done. Please take a look again ;)